src/gfx/process.cpp

```diff
@@ -519,10 +523,12 @@ struct AttrmapEntry {
 	bool yFlip;
 	bool xFlip;
 
-	static constexpr decltype(protoPaletteID) transparent = SIZE_MAX;
+	static constexpr size_t transparent = SIZE_MAX;
```

Comment:
I was using `decltype` so that making the type less wide would trigger a warning. I think it'd be best to use `= -1` instead, if we're going this route?